libs/blocks/section-metadata/section-metadata.js

```diff
@@ -23,7 +23,6 @@ function handleStyle(div, section) {
 export default function init(el) {
   const section = el.closest('.section');
   if (!section) return;
-  section.className = 'section';
```

Comment:
Just want to make sure this is intentional.

Reply:
Yes, this is what we discussed where setting the `className` to `section` overwrites any other classes that already existed. Also, since `section` is queried using the `.section` selector, there's no need to re-add the class.

Reply:
+1